src/components/booking/BookingSuccess.vue

```diff
@@ -0,0 +1,138 @@
+<script setup lang="ts">
+import { computed, watch } from "vue";
+import { storeToRefs } from "pinia";
+import { formattedDateAndTime } from "@/utils/formattedDate.js";
+import { useReservationStore } from "@/stores/reservation.js";
+import LoadingData from "@/components/global/LoadingData.vue";
+import ErrorMessage from "@/components/global/LoadingData.vue";
+import BookingSuccessTitle from "@/components/booking/BookingSuccessTitle.vue";
+
+import BaseButton from "../global/BaseButton.vue";
+
+const reservationStore = useReservationStore();
+const {
+  reservedSeance,
+  reservation,
+  reservationIsLoading,
+  reservationError,
+  getReservationErrorMessage,
+} = storeToRefs(reservationStore);
+
+const getReservationId = computed(() => {
+  return reservedSeance.value?.id;
+});
```

Comment:
Why not simply name it `reservationId`? Computed is not a function in itself so the `get` part is kind of strange 😄 
```suggestion
const reservationId = computed(() => {
  return reservedSeance.value?.id;
});
```